learning/intermediate/session-4.js

```diff
@@ -45,8 +45,24 @@ const person8 = null
 // IF they CAN drink then you will console log out there {toastMessage} AND THEN return TRUE
 // IF they can NOT drink, do nothing and return FALSE
 // keep in mind that it's POSSIBLE for the object to be empty or undefined, and therefore the properties {age} and {toastMessage} may NOT be defined
-function canPersonDrink() {
+function canPersonDrink(person) {
   // fill in the function here
+
+  const checkAge = person.age
+  const checkToastMsg = person.toastMessage
```

Comment:
You don't really need this, but it's fine. Although this isn't actually checking anything, it's simply assigning a property in the person object to a constant. So if anything it should be called _something like_ `personAge` and `personToastMessage`. You're not really doing yourself any favours by naming you variables things that could be confusing.